docs/core/learn-more/authentication.md

```diff
@@ -1,35 +1,97 @@
-# Authentication
+# Authentication and Authorization
 
-## Introduction
+## Authentication
+
+Chain Core allows control of which entities have access to certain features of the system. There are two methods available for limiting access:
+
+1. Access tokens using HTTP Basic Authentication
+2. X.509 client certificates
+
+These authentication objects can be created and managed via the Chain Core Dashboard, SDKs, or the [`corectl`](../reference/corectl) command line tool.
+
+For convenience, in all desktop installations of Chain Core **access from localhost does not require authentication**.
+
+## Authorization
 
 There are two APIs in Chain Core: the **client API** and the **network API**.
 
-The client API is used by the SDKs and the dashboard to communicate with Chain Core. The network API is used by [network operators](blockchain-operators.md).
+The client API is used by the SDKs and the dashboard to communicate with Chain
+Core. The network API is used by [network operators](blockchain-operators.md).
 
-Each API is authenticated using access tokens with HTTP Basic Authentication. 
+There are four policies available to grant an individual authentication method
+access to one or both APIs:
 
-For convenience, **when accessing from localhost, neither API requires authentication**.
+* **client-readwrite**: Full access to the Client API.
+* **client-readonly**: Access to read-only Client endpoints. This is a strict
+subset of the `client-readwrite` policy.
+* **monitoring**: Access to monitoring-specific endpoints. This is a strict
+subset of the `client-readonly` policy.
+* **network**: Access to the Network API.
 
-## Creating access tokens
+## Setting Up
 
-_The instructions in this section require having the Go programming environment installed and the `PATH` variable correctly configured. See [the Chain Core Readme file](https://github.com/chain/chain/blob/main/Readme.md) for details._
+### Mac, Windows and Docker
 
-Both client and network access tokens are created in the dashboard. However, when deploying Chain Core to a non-local environment, you will not be able to access the dashboard, because you will not yet have a client access token. Therefore, you must use the `corectl` command line tool to create your first client access token. After that, you can use that access token to login to the dashboard and create additional access tokens.
+Chain Core Developer Edition is distributed with a flag automatically permitting
+all requests from localhost without configuring an authorization method.
```

Comment:
"For convenience, Chain Core Developer Edition permits all API requests originating from the same host (i.e., localhost) without the need for credentials."